.circleci/config.yml

```diff
@@ -156,11 +156,26 @@ workflows:
   version: 2
   helics_test:
     jobs:
-      - helicsMSan
-      - helicsASan
-      - helicsInstall1
-      - helicsInstall2
-      - helicsgccTSan
+      - helicsMSan:
```

Comment:
any way to make the filters once vs in every job?

Reply:
I tried adding the filters to the workflow, but CircleCI gave errors about it not confirming to their schema.